### PR TITLE
closes #30

### LINKED ISSUE #30: Const correctness
consider making the getters const. 
The functions which are not changing the class should always be declared const
cpustat.hpp // line 18, 20

### PR DESCRIPTION
**Pull request by lmengnr**

### CODE REVIEW

File: src/cpustat.hpp [hunk 1]
```diff
@@ -15,9 +15,9 @@ void getHostname(std::string &hostname) {
 class CpuUsage {
 
 public:
-  double get_curr_val();
+  double get_curr_val() const;
```
Comment (sysarcher) [inline src/cpustat.hpp:18]: this will ensure that `get_curr_val()` does not change any `CpuUsage` instances!

File: src/cpustat.hpp [hunk 2]
```diff
@@ -15,9 +15,9 @@ void getHostname(std::string &hostname) {
...
   const std::string node_id;
-  int get_QOS() { return QOS; };
+  int get_QOS() const { return QOS; };
```
Comment (sysarcher) [inline src/cpustat.hpp:20]: LGTM!

**Review by sysarcher — COMMENTED** [review]
Added my review!!

File: src/cpustat.hpp (hunk 2 above)
Comment (lmengnr) [reply src/cpustat.hpp:20]: thanks

**Review by sysarcher — APPROVED** [review]
Now it looks good

**Pull request merged**